sim/priest/talents.go

```diff
@@ -44,6 +44,39 @@ func (priest *Priest) applyTalents() {
 			},
 		})
 	}
+	
+	if priest.Talents.Enlightenment > 0 {
+		coeff := 0.01 * float64(priest.Talents.Enlightenment)
```

Comment:
You can collapse these 3 parts of enlightenment, so they share the 'if' statement and the coeff value.